### ISSUE TITLE
Update to eclipse 2020/09 platform

### ISSUE THREAD

**Issue by simlei** [jcryptool/core#194]

**Comment by simlei** [comment]
will require to redo the nonautomated part of replacing org.eclipse.help.ui with the [patched version](https://github.com/simlei/org.jcryptool.thirdparty/tree/master/projects/org.jcryptool.thirdparty.platformpatches/org.eclipse.help.ui) that is based currently on 2019/09

**Comment by simlei** [comment]
The code review for the patch I submitted is here: https://git.eclipse.org/r/#/c/157591/

It has seen some progress the last weeks with serious consideration of getting merged thanks to the reviewers. It seems most likely if that's the case, that it will land in early eclipse v. 4.17 i.e. 2020/09

Normally, quarterly releases are being made available in integration builds at the start of that quarter. That means, optimistically we'll see the code in integration builds some time in July 2020.

Updating the platform before that is possible, but requires re-applying the patch onto 2020/03 or /06. It means one of two approaches to be followed:
- manually looking up all of the precise versions that the eclipse components in our target platform depend on. This is, because tycho refuses to accept the patch without a version bump. To do a version bump, I have to know, which version(s) to actually bump. This, I failed to do last time because I found it easier to do method 2 below at some point deep into the process. However, with some information that I was able to reach [here](https://www.eclipse.org/forums/index.php/t/1100313/) and [here](https://www.eclipse.org/forums/index.php/t/1101081/) it should work this time I think. Could be worth the time to do it properly for the insights.
- Patch and build the artifact (org.eclipse.platform.ua) in isolation and trick maven into accepting the patched jar under the same version number as the unpatched one which also is time-consuming and would provide no new insights at all.

With that in mind I am thinking about postponing the target platform until the patched version is officially merged. I do not know of any benefits we get out of 2020/03 over 2019/09 yet -- also, delivering future updates of the eclipse platform over the built-in mechanism is something we have to face in the future in any case.

@grthor FYI, should discuss this in the next meeting
